x-pack/plugins/secops/public/components/timeline/data_providers/empty.tsx

```diff
@@ -54,14 +54,20 @@ const NoWrap = styled.div`
 export const Empty = pure(() => (
   <EmptyContainer className="timeline-drop-area" data-test-subj="empty">
     <NoWrap>
-      <Text color="subdued">{i18n.DROP_ANYTHING}</Text>
+      <Text color="subdued" size="s">
+        {i18n.DROP_ANYTHING}
+      </Text>
       <BadgeHighlighted color="#d9d9d9">{i18n.HIGHLIGHTED}</BadgeHighlighted>
     </NoWrap>
 
     <NoWrap>
-      <Text color="subdued">{i18n.HERE_TO_BUILD_AN}</Text>
+      <Text color="subdued" size="s">
```

Comment:
Above is the `<BadgeHighlighted color="#d9d9d9">` a left over hex color code? Should that change?